src/Stripe.net/Services/Account/StripeAccountAdditionalOwner.cs

```diff
@@ -4,37 +4,37 @@
 
     public class StripeAccountAdditionalOwner : INestedOptions
     {
-        [JsonProperty("[address][city]")]
+        [JsonProperty("address[city]")]
         public string CityOrTown { get; set; }
 
-        [JsonProperty("[address][country]")]
+        [JsonProperty("address[country]")]
         public string Country { get; set; }
 
-        [JsonProperty("[address][line1]")]
+        [JsonProperty("address[line1]")]
         public string Line1 { get; set; }
 
-        [JsonProperty("[address][line2]")]
+        [JsonProperty("address[line2]")]
         public string Line2 { get; set; }
 
-        [JsonProperty("[address][postal_code]")]
+        [JsonProperty("address[postal_code]")]
         public string PostalCode { get; set; }
 
-        [JsonProperty("[address][state]")]
+        [JsonProperty("address[state]")]
         public string State { get; set; }
 
-        [JsonProperty("[dob][day]")]
+        [JsonProperty("dob[day]")]
         public int? BirthDay { get; set; }
 
-        [JsonProperty("[dob][month]")]
+        [JsonProperty("dob[month]")]
         public int? BirthMonth { get; set; }
 
-        [JsonProperty("[dob][year]")]
+        [JsonProperty("dob[year]")]
```

Comment:
Ideally, all nested params should have their own class. Did you mention this one specifically because all 3 `day/month/year` parameters are required? If so, then yes, we could make the nested class optional but have all 3 parameters within the class be required.